.gitignore

```diff
@@ -9,6 +9,7 @@
 .metadata/
 .project
 .settings/
+/journal/
```

Comment:
On a related note, we should use the leading '/' anywhere where we indeed mean the top-level tachyon directory (e.g. /conf/tachyon-env.sh). I would just add a TODO for this.